pantheon/drupal7/lockr_pantheon/lockr_pantheon.install

```diff
@@ -0,0 +1,46 @@
+<?php
+
+/**
+ * @file
+ * Install, uninstall, and update functions for lockr_pantheon.
+ */
+
+/**
+ * Implements hook_uninstall().
+ *
+ * Delete any keys that use Lockr as the key provider.
+ */
+function lockr_pantheon_uninstall() {
+  // Load the key configurations.
+  $configs = db_query("SELECT * FROM {key_config} ORDER BY label ASC")->fetchAllAssoc('id', PDO::FETCH_ASSOC);
```

Comment:
For the future, we might want to wrap this query (as well as the loop below) into a function for key like `key_get_keys_for_provider($provider_name)` so it's easier for providers to implement this kind of feature.